_posts/2024-01-20-jan-meetup.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: post
+title:  "January 2024 Meetup"
+authors: 
+  - nvs-abhilash
+description: "January Meetup at Microsoft Reactor Bengaluru"
+categories: [ meetup, talks]
+featured: true
+---
+
+BangPypers commenced its 2024 journey with the first meetup of the month on January 20, 2024, at Microsoft Reactor, Bangalore. The meetup was attended by 25 people who were a mix of new faces and returning members. We had a diverse lineup of three talks involving [MicroPython](https://micropython.org), [Poetry](https://python-poetry.org), and [Dunderhell](https://github.com/tusharsadhwani/dunderhell) (writing Python code using only dunders).
+
+We started with [Md Moinuddin](https://twitter.com/noobyco) presenting the talk [MicroPython - Python for Microcontrollers](https://github.com/bangpypers/meetup-talks/issues/20). Moinuddin provided an introduction and history of MicroPython, followed by a short demo on how to execute Python code on a microcontroller, and explained his project [Gamex](https://github.com/noobyco/gamex). Finally, he explained some differences (and similarities) between CPython and MicroPython to answer the question, "Will my Python code run on MicroPython?"
+
+![Md Moinuddin presenting MicroPython - Python for Microcontrollers](<image_link> "Md Moinuddin presenting MicroPython - Python for Microcontrollers")
+
+Our second talk was presented by [Pratik Pathak](https://twitter.com/zpratikpathak), where he wanted to convince us that ["Time to Ditch requirements.txt and VENV, Start using Poetry now"](https://github.com/bangpypers/meetup-talks/issues/21). Pathak started by demonstrating an issue on why using a simple `requirements.txt` file for our Python projects is not enough. He then explained how using [Poetry](https://python-poetry.org) can simplify Python's packaging and dependency management.
```

Comment:
Maybe we don't say "convince us". Cause it leaves the question if we were convinced. 😅

Reply:
fixed.